### DISCUSSION TITLE
can support multi static file path？

### DISCUSSION THREAD

**Discussion by fengshaoxie** [cesanta/mongoose#1808] [Ask questions, get answers] [answered]
- My goal is: add multi static file path
- My actions were: _describe_
- My expectation was: _describe_
- The result I saw: _provide logs or description_
- My question is: _your question_

# Environment

- mongoose version: 6.18
- such as
![image](https://user-images.githubusercontent.com/37693004/196913922-c88568a8-0b53-40b6-9f6b-6475d4ccfe2e.png)

**Comment by scaprile** [comment, marked as answer]
Match the URI https://github.com/cesanta/mongoose/blob/6.18/examples/restful_server/restful_server.c and serve the dir https://github.com/cesanta/mongoose/blob/6.18/docs/http/files.md

**Comment by cpq** [comment]
@fengshaoxie https://mongoose.ws/documentation/#mg_http_serve_dir and look at the example
